common/libs/values.ts

```diff
@@ -14,3 +14,15 @@ export function valueToHex(value: Ether): string {
   // Finally, hex it up!
   return `0x${wei.toString(16)}`;
 }
+
+export function padLeftEven(hex) {
+  return hex.length % 2 !== 0 ? '0' + hex : hex;
+}
+
+export function sanitizeHex(hex) {
+  hex = hex.substring(0, 2) === '0x' ? hex.substring(2) : hex;
```

Comment:
Should probably re-name this and make it a constant, mutating parameters isnt a good idea

Reply:
This is all lifted from V3, I but I agree with you. I'll make these changes and note that we're diverging from V3.

Reply:
On second thought, I'm going to leave a TODO and circle back to this once we have a more comprehensive test suite.